### DISCUSSION TITLE
Loading a lot of new maps at once can cause this

### DISCUSSION THREAD

**Discussion by stevefan1999-personal** [ppy/osu#23270] [Q&A] [answered]
Just want to be sure we don't have an existing issue, but the symptom looks like this:
![image](https://user-images.githubusercontent.com/29133953/233651376-b0c43fe3-d833-4a70-b40c-89b5aee1a744.png)

I migrated a lot of maps from my old drive into the new one. There are about 1000 songs to be imported at once. Seems like the beatmap processor caught a stroke and did not finish processing. Far as I can tell, all the files and songs data (including BGA, music and hit sounds) are intact and readable. I think it maybe related to Data/bt? 

Reproduction: I replicated the old Songs folder into a clone, emptied the old one and cut some cloned beatmap into the new empty Songs folder. This problem can be consistently triggered even with 200 maps to load at once.

Also I can click update to latest version and then get the map back partially. But only to that specific diff however, so far in my testing (so you can "recover" Extra but not the rest like Insane or Hard).

Repairing osu does not help, and I believe we might need to throttle the degree of parallelism for beatmap processor to prevent a potential race condition (alas, at the cost of performance) which I believe is the case that corrupted the index database. If this problem truly is never seen before, I would (or any dev, much appreciated) like to escalate this into an issue.

**Comment by stevefan1999-personal** [comment]
Well...let's say forget about this. Since I was playing on Osu Classic but this is the discussion panel for Osu Lazer, this means I posted it to the wrong place despite they are oh so similar. So, would someone point me to the right place instead?

**Reply by sineplusx** [reply, marked as answer]
The repo to report issues for the stable release is [osu-stable-issues](https://github.com/ppy/osu-stable-issues)
